tools/generate-wire.py

```diff
@@ -832,28 +832,28 @@ def print_struct(self):
 
 tlv__type_impl_towire_fields = """\tif ({tlv_name}->{name}) {{
 \t\ttlv_msg = tal_arr(ctx, u8, 0);
-\t\ttowire_u16(p, {enum});
+\t\ttowire_u8(p, {enum});
```

Comment:
Except I was wrong and it (and type) should be varint :)

Reply:
updated!!